### ISSUE TITLE
[iOS] prompt for biometrics delayed when calling getGenericPassword

### ISSUE THREAD

**Issue by allisonkbarr** [oblador/react-native-keychain#256]
Since updating my iOS to 13.1.2 yesterday, I have observed an issue with the getGenericPassword call in my app. The prompt to the user for biometrics does not appear, or sometimes appears only after a several second wait, or sometimes only appears when the user starts taking another action to navigate away (e.g. double pressing the home button).

Device I am testing on is iPhone 6S with iOS version 13.1.2.

react-native-keychain version: 3.1.3

setGenericPassword call:
```
await Keychain.setGenericPassword(
  credentials.emailAddress,
  credentials.password,
  {
    service: KEYCHAIN_SERVICE_ID,
    accessControl: Keychain.ACCESS_CONTROL.BIOMETRY_ANY,
    accessible: Keychain.ACCESSIBLE.WHEN_PASSCODE_SET_THIS_DEVICE_ONLY,
    securityLevel: 'SecureSoftware'
  } as any
);
```
getGenericPassword call:
```
const result = await Keychain.getGenericPassword({
  service: KEYCHAIN_SERVICE_ID,
  authenticationPrompt: 'Authenticate to complete setup'
});
```
react-native info:
```
System:
    OS: macOS 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i5-8257U CPU @ 1.40GHz
    Memory: 217.37 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.16.3 - ~/.nvm/versions/node/v10.16.3/bin/node
    npm: 6.9.0 - ~/.nvm/versions/node/v10.16.3/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.0, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
  IDEs:
    Xcode: 11.0/11A420a - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6
    react-native: 0.60.0 => 0.60.0
  npmGlobalPackages:
    react-native-cli: 2.0.1
```

**Comment by aeirola** [comment]
This seems to be related to an issue with iOS itself in versions 13.0 and 13.1. There is some discussion around the issue at https://forums.developer.apple.com/thread/121149, with a possible workaround.

The bug seems to be fixed in iOS 13.2 beta 1 though, which will probably be released by Apple quite promptly due to the large amount of issues still present in 13.1.

Since the issue will only affect a small number of iOS versions, I don't think it is worth the effort of trying to work around it in this library. That said, I think it might be possible to perform the same workaround in your code, by calling `Keychain.getGenericPassword` twice in a row for devices running iOS 13.0 and 13.1. Unfortunately I don't have time to test that out right now, but in case you do go ahead and attempt that, please share your results here.

**Comment by allisonkbarr** [comment]
I've confirmed I see this same issue in other apps on iOS 13.1. Since we haven't released the app in question yet, I'll hold on trying any workarounds and anticipate the fix to come from Apple with 13.2. Thanks!
